qa/tools/apex_algorithm_qa_tools/test_metrics.py

```diff
@@ -0,0 +1,126 @@
+"""
```

Comment:
`test_` is kind of a reserved prefix in pytest, 
pick something better, with less risk of confusion/collision?

Reply:
renamed to "track_metrics"